js/ui/panel.js

```diff
@@ -1816,10 +1817,45 @@ PanelContextMenu.prototype = {
             confirm.open();
         });
 
+        menu.troubleshootItem.menu.addMenuItem(new PopupMenu.PopupSeparatorMenuItem());
+
+        // Add the System Monitor launcher to the troubleshoot menu
+        let systemMonitorInfo = this._findFirstSystemMonitor();
+        if (systemMonitorInfo) {
+            menu.troubleshootItem.menu.addAction(_(systemMonitorInfo.name), function(event) {
+                let settingsLauncher = new SettingsLauncher(systemMonitorInfo.name, "", "", systemMonitorInfo.command);
+                settingsLauncher.activate();
+            });
+        }
+
         menu.addMenuItem(new PopupMenu.PopupSeparatorMenuItem()); // separator line
         menu.addMenuItem(menu.troubleshootItem);
 
-        this.addMenuItem(new SettingsLauncher(_("System Settings"), "", "preferences-desktop"));
+        this.addMenuItem(new SettingsLauncher(_("System Settings"), "", "system-run", "cinnamon-settings"));
```

Comment:
Although working with the other icon themes I tested, the system-run icon presents something like a play icon for the breeze icon theme, so it might not be the best suited for System Settings there.

Reply:
The entry is in the Troubleshooting submenu now, so the icon was removed also.